DependencyInjection/Configuration.php

```diff
@@ -90,6 +90,20 @@ public function getConfigTreeBuilder()
                         ->end()
                     ->end()
                 ->end()
+                ->arrayNode('discovery')
+                    ->addDefaultsIfNotSet()
+                    ->info('Control what clients should be found by the discovery.')
+                    ->children()
+                        ->scalarNode('client')
+                            ->defaultValue('auto')
+                            ->info('Set to "auto" to see auto discovered client in the web profiler. If provided a service id for a client then this client will be found by auto discovery.')
+                        ->end()
+                        ->scalarNode('async_client')
+                            ->defaultNull()
```

Comment:
okay. but then lets change the info to mention that you should set this to auto if you know that you have an async client available.